linera-execution/tests/fee_consumption.rs

```diff
@@ -150,6 +150,7 @@ async fn test_fee_consumption(
         maximum_executed_block_size: 37,
         maximum_blob_size: 41,
         maximum_bytecode_size: 43,
+        maximum_block_proposal_size: 47,
         maximum_bytes_read_per_block: 47,
         maximum_bytes_written_per_block: 53,
```

Comment:
(Let's continue the tradition!)

```suggestion
        maximum_bytes_read_per_block: 53,
        maximum_bytes_written_per_block: 59,
```